src/test/java/com/google/api/generator/engine/ast/CastExprTest.java

```diff
@@ -73,6 +73,49 @@ public void validCastExpr_basicPrimitiveWidening() {
 
   }
 
+  @Test
+  public void validCastExpr_castPrimitiveToBoxedType() {
+    PrimitiveValue intValue = PrimitiveValue.builder().setValue("3").setType(TypeNode.INT).build();
+    ValueExpr valueExpr = ValueExpr.withValue(intValue);
+    CastExpr expr = CastExpr.builder().setType(TypeNode.INT_OBJECT).setExpr(valueExpr).build();
```

Comment:
Could we remove `CastExpr expr` since it's unused? Same below.

Reply:
done.